src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -1920,9 +1903,6 @@ private void Model_PropertyChanged(object sender, PropertyChangedEventArgs e)
                 case nameof(EnableTSplineIsChecked):
                     description = Res.ResourceManager.GetString(nameof(Res.PreferencesViewEnableTSplineNodes), System.Globalization.CultureInfo.InvariantCulture);
                     goto default;
```

Comment:
I think we can remove the Tspline related stuff too while we are at it. I'm not sure why it's still there.